nautobot/circuits/migrations/0008_auto_tags.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.14 on 2022-03-19 23:21
+
+from django.db import migrations
+import nautobot.extras.models.tags
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0030_webhook_alter_unique_together"),
+        ("circuits", "0007_circuitterminations_primary_model"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="circuit",
+            name="tags",
+            field=nautobot.extras.models.tags.TaggableManagerField(through="extras.TaggedItem", to="extras.Tag"),
```

Comment:
Need to consider how this may affect plugins -- if a plugin's models continue to use the base `TaggableManager` class for backwards-compatibility with earlier Nautobot versions, how does that affect the UI/UX of using the updated tags with those models? Does it mean that plugin models can be tagged with tags that don't actually apply to their content-types? Or does trying to use a mismatched tag result in a user-friendly error message?

Reply:
I confirmed that the migrations generated by this diff do not actually result in any SQL changes, it's just effectively a change to the underlying manager field class for the sake of consistency in the migrations and nothing more:

```
$ nautobot-server sqlmigrate circuits 0008
BEGIN;
--
-- Alter field tags on circuit
--
--
-- Alter field tags on circuittermination
--
--
-- Alter field tags on provider
--
--
-- Alter field tags on providernetwork
--
COMMIT;
```